indra/cmake/00-Common.cmake

```diff
@@ -198,7 +198,7 @@ if (LINUX OR DARWIN)
   list(APPEND GCC_WARNINGS -Wno-reorder -Wno-non-virtual-dtor )
 
   if(LINUX)
-    list(APPEND GCC_WARNINGS -Wno-maybe-uninitialized -Wno-misleading-indentation -Wno-stringop-truncation -Wno-unused-value )
+    list(APPEND GCC_WARNINGS -Wno-maybe-uninitialized -Wno-dangling-pointer )
```

Comment:
Where did you encounter this issue. I have not been able to reproduce with
`gcc version 13.2.1 20240210 (Gentoo 13.2.1_p20240210 p14)`

Reply:
The problem occurs when bumping boost to 1.84.
It does not occur now and can be removed, but it will occur if you update to 1.84 in the future, so you will need to fix it anyway.

Reply:
Got it. We shall look at it again when then time comes. Maybe there's another way to get rid of the warning without having to disable it on the whole code base